py/generate.py

```diff
@@ -988,7 +988,7 @@ def main(browser_protocol_path, js_protocol_path, output_path):
         browser_protocol_path,
         js_protocol_path,
     ]
-    output_path.mkdir(parents=True)
+    output_path.mkdir(parents=True, exist_ok=True)
```

Comment:
I think you don't need the mkdir. Declare directory should have done it for you.

Reply:
removed mkdir